src/metabase/server/middleware/log.clj

```diff
@@ -194,7 +194,7 @@
   ;; don't log calls to /health or /util/logs because they clutter up the logs (especially the window in admin) with
```

Comment:
@paoliniluis please note the health endpoint logging was disabled on purpose.

Reply:
Yup! Is there a reason why?

Reply:
Just log spam, as per the comment. The logs endpoint still has logging for the call disabled. Other than that one, logging seems to be applied to all endpoints.